app/webpacker/components/Panel/pages/MergeUsersPage/api/getUserDetails.js

```diff
@@ -0,0 +1,14 @@
+import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../../../lib/requests/routes.js.erb';
+
+export default async function getUserDetails(userId) {
+  const { data } = await fetchJsonOrError(
+    viewUrls.users.show(userId),
+    {
+      headers: {
+        Accept: 'application/json',
+      },
+    },
```

Comment:
Isn't this automatically part of `fetchJsonOrError`?!

Reply:
Not sure, but without this, it is working now, so removed. I cannot recall exactly why I had this earlier, mostly a miss. Anyway thanks.